modules/nw-dual-stack-convert.adoc

```diff
@@ -0,0 +1,81 @@
+[id="nw-dual-stack-convert_{context}"]
+= Converting to a dual network stack cluster network
+
+As a cluster administrator, you can convert your single stack cluster network to a dual-stack cluster network.
+
+[NOTE]
+====
+After converting to dual-stack networking only newly created pods are assigned IPv6 addresses. Any pods created before the conversion must be recreated to receive an IPv6 address.
+====
+
+.Prerequisites
+
+* You installed the OpenShift CLI (`oc`).
+* You are logged in to the cluster with a user with `cluster-admin` privileges.
+* Your cluster is using the OVN-Kubernetes cluster network provider.
```

Comment:
This is a nit.  My pref is "Your cluster uses..." but what you have is likely consistent with other files.